doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1102,7 +1102,9 @@ Other libraries
 Security libraries
 ------------------
 
-|no_changes_yet_note|
+* :ref:`trusted_storage_readme`:
+
+  * Added support for the ZMS file subsystem for devices that contain RRAM, such as the nRF54L Series devices.
```

Comment:
I wouldn't call this a file subsystem or even file system... Also, it's not just for the 54L devices that it's available. Maybe change this to just:
```suggestion
  * Added support for ZMS as an alternative to NVS.
```
(edit: I noticed `file system` was already used for NVS. Maybe it's a correct terminology? Anyway, we can stick to that. But this sentence still needs an update anyway.)